e2e/docker/compose.yaml

```diff
@@ -0,0 +1,34 @@
+name: flwr-containers-e2e-test
+
+services:
+  # create a SuperLink service
+  superlink:
+    image: docker.io/flwr/superlink:1.9.0.dev20240513
```

Comment:
```suggestion
    image: docker.io/flwr/superlink:nightly
```

Reply:
I wonder if we should better point it to the latest nightly. If so, let's change it in the `ServerApp` and `ClientApp` Dockerfile too.

Reply:
That's a good point. I'll update it.